### PR TITLE
Feat/ll 61

### PR DESCRIPTION
**Pull request by oleksandrtarapata**
Redesigned market cards
Added multi token support by market cards (extended mock data)
Added denomination for liquidity and volume at market card
Brushed components touched

task [LL-61](https://www.notion.so/limitlesslabs/7e5052e4a3a6414da3d2b6ddf8679519?v=1d97039d06fb4c42ad202538d745e1bf&p=79b8c0e8c8e6459fab887525c8700d3f&pm=s)

### CODE REVIEW

**Comment by valentynkvasov** [comment]
image

@oleksandrtarapata Could you make WETH denomination looks like 0.9,454 -> 0.9454 and others

**Comment by valentynkvasov** [comment]
@oleksandrtarapata good job, it was quick!

**Comment by oleksandrtarapata** [comment]
> image
> @oleksandrtarapata Could you make WETH denomination looks like 0.9,454 -> 0.9454 and others

updated

File: src/components/markets/MarketCardUserActions.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const MarketCardUserActions = ({
         p={1}
         onClick={() => window.open(shareLinks.castURI, '_blank', 'noopener')}
       >
-        <Image src={'/assets/images/farcaster.png'} h={'22px'} alt={'Farcaster'} />
+        <Image src={'/assets/images/warcaster.svg'} h={'16px'} w={'16px'} alt={'warcaster'} />
```
Comment (maxschnaider) [inline src/components/markets/MarketCardUserActions.tsx:59]: typo: warcaster -> warpcast

Comment (oleksandrtarapata) [reply src/components/markets/MarketCardUserActions.tsx]: > warpcast

updated

**Pull request merged**